### PR TITLE
Replaced writeouts to System.out with logging

### PR DESCRIPTION
**Pull request by jreeseUA**
Replaced writeouts to System.out with logging - All logging to system console was modified to write to the selected log file.

### CODE REVIEW

File: src/main/java/ua/utility/kfsdbupgrade/App.java [hunk 1]
```diff
@@ -120,6 +120,7 @@ public App(String propertyFileName, boolean ingestWorkflow) {
             }
         } else {
             System.out.println("invalid properties file: " + propertyFileName);
```
Comment (Perrinek) [inline src/main/java/ua/utility/kfsdbupgrade/App.java]: This line should be removed.

**Comment by Perrinek** [comment]
If methods writeHeader1 and writeHeader2 are no longer in use, they should be removed.

**Comment by Perrinek** [comment]
Line 94: System.out.println("usage: java -Xmx500m -jar kfsdbupgrade.jar <property-file-path>");
Needs to be replaced with logging.

**Comment by Perrinek** [comment]
If methods writeOut(Exception ex) and writeOut(String msg) are no longer in use, they should be removed.

**Comment by Perrinek** [comment]
Initial review complete. Please see my comments. Please include the Jira number at the beginning of each commit message. Where method names have changed, please update references in comments.

**Comment by jreeseUA** [comment]
All the changes in this UAF-2786 as well as UAF-2786 code review recommendations above have been included in UAF-2826 which was reviewed and merged. Hence closing this request now. Thanks.

**Pull request closed without merging**